proto/interchain_security/ccv/consumer/v1/genesis.proto

```diff
@@ -55,4 +55,4 @@ message HeightToValsetUpdateID {
 
 // OutstandingDowntime defines the genesis information for each validator
 // flagged with an outstanding downtime slashing.
-message OutstandingDowntime { string validator_consensus_address = 1; }
\ No newline at end of file
+message OutstandingDowntime { string validator_consensus_address = 1; }
```

Comment:
What changed here?

Reply:
nothing, prob my editor just formatting it.